lua/autorun/client/cl_chip_lister.lua

```diff
@@ -0,0 +1,308 @@
+include( "cfc_chip_lister/shared/sh_chip_lister.lua" )
+
+local ID_WORLD = "[WORLD]"
+local CPUS_FORMAT = "%05d"
+local STR_GLOBAL = "Overall Total CPUs: "
+local STR_TOTAL = "Total: "
+local STR_MICROSECONDS = utf8.char( 181 ) .. "s"
+local STR_TITLE = "-----E2/SF Lister-----"
+local STR_TOGGLE = "(Press " .. string.upper( input.LookupBinding( "+use" ) or "e" ) .. "/use to toggle)"
+local STR_ENABLE = "Press " .. string.upper( input.LookupBinding( "+use" ) or "e" ) .. "/use to turn on"
+local STR_WAITING = "Waiting for next update from the server..."
+local RENDER_TARGET_NAME = "cfc_chiplister_rt"
+local FONT_NAME = "CFC_ChipLister_Font"
+local FONT_SIZE = 30
+local MAX_ELEMENTS = 30
+local SCREEN_SIZE = 1024
+local SCREEN_SIZE_HALF = SCREEN_SIZE / 2
+local TOGGLE_DIST = 1500
+local COLOR_BACKGROUND = Color( 0, 0, 0, 255 )
+local COLOR_DIVIDER = Color( 255, 255, 255, 255 )
+local COLOR_TEXT = Color( 255, 255, 255, 255 )
+local COLOR_TEXT_FADED
+local COLOR_MICROS
+local COLOR_WORLD = Color( 150, 120, 120, 255 )
+local HSV_FADE_OFFSET = Vector( 0, 0, -0.65 )
+local HSV_FADE_MICROS_OFFSET = Vector( 0, 0, -0.25 )
```

Comment:
Breaking these up into chunks might make it a little more parse-able.

Honestly I think a lot of these don't need to be constants. They _can_ be, but I don't think it adds enough benefit to justify making the top of the file so unwelcoming.

Up to you, though
```suggestion
local ID_WORLD = "[WORLD]"
local CPUS_FORMAT = "%05d"

local RENDER_TARGET_NAME = "cfc_chiplister_rt"
local STR_GLOBAL = "Overall Total CPUs: "
local STR_TOTAL = "Total: "
local STR_MICROSECONDS = utf8.char( 181 ) .. "s"
local STR_TITLE = "-----E2/SF Lister-----"
local STR_TOGGLE = "(Press " .. string.upper( input.LookupBinding( "+use" ) or "e" ) .. "/use to toggle)"
local STR_ENABLE = "Press " .. string.upper( input.LookupBinding( "+use" ) or "e" ) .. "/use to turn on"
local STR_WAITING = "Waiting for next update from the server..."

local FONT_NAME = "CFC_ChipLister_Font"
local FONT_SIZE = 30

local MAX_ELEMENTS = 30
local TOGGLE_DIST = 1500
local SCREEN_SIZE = 1024
local SCREEN_SIZE_HALF = SCREEN_SIZE / 2

local COLOR_BACKGROUND = Color( 0, 0, 0, 255 )
local COLOR_DIVIDER = Color( 255, 255, 255, 255 )
local COLOR_TEXT = Color( 255, 255, 255, 255 )
local COLOR_TEXT_FADED
local COLOR_MICROS
local COLOR_WORLD = Color( 150, 120, 120, 255 )

local HSV_FADE_OFFSET = Vector( 0, 0, -0.65 )
local HSV_FADE_MICROS_OFFSET = Vector( 0, 0, -0.25 )
```